components/PoisList/PoisTable.vue

```diff
@@ -171,11 +177,11 @@ function getContext(key: string) {
         <header class="d-flex align-center pa-2" :style="{ flexDirection: device.smallScreen ? 'column' : 'row', gap: '8px', background: '#eeeeee' }">
           <h1 class="d-flex align-center print:tw-pb-4" :style="{ marginRight: device.smallScreen ? 'unset' : 'auto' }">
             <TeritorioIconBadge
-              :color-fill="category.category.color_fill"
-              :picto="category.category.icon"
+              :color-fill="category?.category.color_fill || ''"
+              :picto="category?.category.icon || ''"
               size="xl"
             />
-            {{ category.category.name.fr }}
+            {{ category?.category.name.fr }}
```

Comment:
Make this part conditioned by a `v-if="category"` ?

Reply:
I didn't in ordre to keep 'empty' UI and make the use of default props instead.
I give a 'more' consistent UI to user's eye and reduce Layout Shift as well.

It was also a thinking about whether to have en ampty page or page with empty state UI.